app/assets/stylesheets/custom/deck.css.scss

```diff
@@ -248,3 +248,15 @@ th .header{
 .archived{
   display: none;
 }
+#notes{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space